pyminhash/pyminhash.py

```diff
@@ -148,7 +149,15 @@ def fit_predict(self, df: pd.DataFrame, col_name: str) -> pd.DataFrame:
             Pandas dataframe containing pairs of (partial) matches
 
         """
-        df_ = df[[col_name]].drop_duplicates().copy()
+        df_ = df[[col_name]].copy()
+        # get boolean series indicating if a value is a duplicate
+        duplicates = df_.duplicated()
+        # check if there is any duplicate value in the series
+        if duplicates.any():
+            duplicate_count = duplicates.value_counts().loc[True]
+            message = "There are {} duplicates in the dataset for column '{}', consider removing duplicates first".format(
+                duplicate_count, col_name)
+            warnings.warn(message)
```

Comment:
Could you please create the message inside the warn() call and use f-string formatting?

Reply:
Ofcourse! However I purposley did it this way as right now the warning looks like this in the console:
`/**/pyminhash/pyminhash.py:158: UserWarning: There are 407 duplicates in the dataset for column 'title', consider removing duplicates first
  warnings.warn(message)`
  
  Whereas with f-string formatting and within the warning() it looks like this in the console:
  `/**/pyminhash/pyminhash.py:157: UserWarning: There are 407 duplicates in the dataset for column 'title', consider removing duplicates first
  warnings.warn(f"There are {duplicate_count} duplicates in the dataset for column '{col_name}', consider removing duplicates first")`
  
  Let me know which one you prefer, I'm fine with either of them!

Reply:
that's strange. i see it as well. then i would define the message first as you did, but then with f-string formatting please

Reply:
Updated!